### ISSUE TITLE
bug(next): `<DropdownMenu.Content/>` not wrapped in a portal

### ISSUE THREAD

**Issue by ieedan** [huntabyte/shadcn-svelte#1501]
### Describe the bug

Right now dropdown-menu-content is not rendered in a portal. Easy fix but I think this used to work differently before?

dropdown-menu-content.svelte
```svelte
<DropdownMenuPrimitive.Content
	{transition}
	{transitionConfig}
	{sideOffset}
	class={cn(
		"bg-popover text-popover-foreground z-50 min-w-[8rem] rounded-md border p-1 shadow-md focus:outline-none",
		className
	)}
	{...$$restProps}
	on:keydown
>
	<slot />
</DropdownMenuPrimitive.Content>
```

### Reproduction

N/A

### System Info

```bash
N/A
```

### Severity

annoyance

**Comment by huntabyte** [comment]
Is this Svelte 4 code? If so, there isn't a separate `Portal` component for dropdown menu in the Svelte 4 version of Bits, it's baked in via the `portal` prop.

**Comment by ieedan** [comment]
Sorry I wasn't on next when I copied that code. 🤦

Yes it's the svelte 5 code.

<script lang="ts">
	import { cn } from "$lib/utils.js";
	import { DropdownMenu as DropdownMenuPrimitive } from "bits-ui";

	let {
		ref = $bindable(null),
		sideOffset = 4,
		class: className,
		...restProps
	}: DropdownMenuPrimitive.ContentProps = $props();
</script>

<DropdownMenuPrimitive.Content
	bind:ref
	{sideOffset}
	class={cn(
		"bg-popover text-popover-foreground data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2 z-50 min-w-[8rem] overflow-hidden rounded-md border p-1 shadow-md outline-none",
		className
	)}
	{...restProps}
/>

**Comment by huntabyte** [comment]
I could have sworn this was wrapped in `DropdownMenuPrimitive.Portal`, either way it should be, nice catch!

**Comment by ieedan** [comment]
It looks like this is actually slotted to be fixed in #1482

**Comment by huntabyte** [comment]
Thanks for bringing the issue to attention!
